src/InputEntry.js

```diff
@@ -1,10 +1,8 @@
 import React from 'react';
 import { Field } from 'react-final-form';
 
-const InputEntry = ( {label, type, name, value} ) => 
-	<div className="form-group">
-		<label>{label}</label>
-	    <Field name={name} component="input" min={type==='number'&&1} value={value&&value} type={type} className={`form-control col-${type==="date" ? 4: type==="number" ? 2: 12}`} />
-	</div>
+const InputEntry = ( {type, name, value, placeholder, min, textChanged} ) => 
+	<Field name={name&&name} component="input" value={value&&value} type={type&&type}
```

Comment:
```
<Field name={name&&name} component="input" value={value&&value} type={type&&type}
<label>{label}</label>  placeholder={placeholder&&placeholder} min={min&&min} className="text_input" onChange={textChanged} />
```

You can refactor this block of code to use default props so you dont check if a value is available before passing it as a prop, within the prop.